### PR TITLE
[ticket/13622] Allow changing how get_visibility_sql() works

### PR DESCRIPTION
**Pull request by brunoais**
https://tracker.phpbb.com/browse/PHPBB3-13622

PHPBB3-13622

### CODE REVIEW

File: phpBB/phpbb/content_visibility.php [hunk 1]
```diff
@@ -143,12 +143,42 @@ public function get_count($mode, $data, $forum_id)
 	*/
 	public function get_visibility_sql($mode, $forum_id, $table_alias = '')
 	{
+		$where_sql = '';
+
+		$get_visibility_sql_overwrite = false;
+		/**
```
Comment (marc1706) [inline phpBB/phpbb/content_visibility.php]: Please add a space before this.

Comment (brunoais) [inline phpBB/phpbb/content_visibility.php]: Added a new line.

Comment (marc1706) [inline phpBB/phpbb/content_visibility.php]: Which is what I meant, thanks. ;)

**Comment by marc1706** [comment]
@nickvergessen comments?

**Comment by nickvergessen** [comment]
I guess its fine?

**Comment by nickvergessen** [comment]
There is already an event with the same name and same changes.
Only the variable comments are different. So I think this is not needed anymore?

**Pull request closed without merging**